importers/data_sources/importer.py

```diff
@@ -0,0 +1,209 @@
+"""
```

Comment:
I started with the only other existing importer in the codebase and stripped out all of the unnecessary stuff, adding back in what was needed. If it were written from scratch (I couldn't find docs for this), it may have been simpler.

Reply:
There aren't really any standardised importers -- it's all been somewhat ad hoc.